src/components/navigation.js

```diff
@@ -1,17 +1,19 @@
 import { Link } from 'react-router-dom';
+import { IoIosPerson } from 'react-icons/io';
 
 const Navigation = () => (
   <div className="bar">
     <nav className="navbar">
-      <h1>Bookstore Ikoote</h1>
+      <h1>Bookstore</h1>
```

Comment:
- [ ] Kindly make this `Bookstore CMS` as seen in the design :wink: So your design looks exactly like the given design :100: